### PR TITLE
KFSPTS-30923: Make IWant set-as-default checkbox and reset-initiator-defaults button usable only during initial creation.

### PR DESCRIPTION
**Pull request by nkimble**

### CODE REVIEW

**Review by jhulslander — CHANGES_REQUESTED** [review]
Changes look good, I found a few things to consider

File: src/main/webapp/WEB-INF/tags/module/purap/iWantCustomerData.tag [hunk 1]
```diff
@@ -60,15 +62,15 @@
                         <kul:htmlControlAttribute
                                 attributeEntry="${documentAttributes.useCollegeAndDepartmentAsDefault}"
                                 property="document.useCollegeAndDepartmentAsDefault"
-                                readOnly="${not fullEntryMode}"
+                                readOnly="${not userMatchesInitiator or not docIsInitiatedOrSaved}"
```
[the remark this answers is not in the document]
Comment (nkimble) [reply src/main/webapp/WEB-INF/tags/module/purap/iWantCustomerData.tag:65]: No, I do not believe that fullEntryMode should be included in the read only logic.

Per Cathy: The "Set as Default" check box AND "Reset Initiator Defaults" button should be usable ONLY during initial IWant document creation by the initiator of the document.

fullEntryMode is true/false based on the value from KualiForm.documentActions[KRADConstants.KUALI_ACTION_CAN_EDIT]. 
My local testing has shown that edit mode allows an approver to also have access to these buttons (which is not wanted) as it is less restrictive than what is being implemented by these changes

**Review by nkimble — COMMENTED** [review]
Updates have been made. This is ready to be reviewed again.

File: src/main/webapp/WEB-INF/tags/module/purap/iWantCustomerData.tag (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/webapp/WEB-INF/tags/module/purap/iWantCustomerData.tag:65]: sorry, what I was trying to suggest is should the items be available if the document is in full entry mode AND the user is the initiator AND the document is in save/initiated status

**Comment by jhulslander** [comment]
Changes look good, I just had one clarification question, I wasn't sure if my original comment conveyed the idea I was trying to present.

**Review by jhulslander — APPROVED** [review]
Nancy and I spoke and code as is seems to be working as needed and can proceed to kfs-dev for validation

**Pull request merged**